resources/assets/coffee/react/beatmapset-page/score-top.coffee

```diff
@@ -25,68 +25,69 @@ BeatmapsetPage.ScoreTop = (props) ->
+          a
+            className: "#{bn}__username"
+            href: laroute.route 'users.show', user: props.score.user.id
+            props.score.user.username
 
-    div className: "#{bn}__section #{bn}__section--bottom",
-      div className: "#{bn}__stats-top",
-        div className: "#{bn}__rank",
-          div className: "badge-rank badge-rank--medium badge-rank--#{props.score.rank}"
+          el FlagCountry,
+            country: props.countries[props.score.user.country_code]
+            classModifiers: ['scoreboard', 'small-box']
 
+      div className: "#{bn}__wrapping-container #{bn}__wrapping-container--right",
         div className: "#{bn}__stats",
           div className: "#{bn}__stat",
-            div className: "#{bn}__stat-header",
+            div className: "#{bn}__stat-header #{bn}__stat-header--wider",
               osu.trans 'beatmapsets.show.scoreboard.stats.score'
             div className: "#{bn}__stat-value #{bn}__stat-value--score",
               props.score.score.toLocaleString()
 
+        div className: "#{bn}__stats",
           div className: "#{bn}__stat",
-            div className: "#{bn}__stat-header",
+            div className: "#{bn}__stat-header #{bn}__stat-header--wider",
               osu.trans 'beatmapsets.show.scoreboard.stats.accuracy'
             div className: "#{bn}__stat-value #{bn}__stat-value--score",
               "#{_.round props.score.accuracy * 100, 2}%"
 
-          div className: "#{bn}__stat hidden-xs",
-            div className: "#{bn}__stat-header",
+          div className: "#{bn}__stat",
+            div className: "#{bn}__stat-header #{bn}__stat-header--wider",
               osu.trans 'beatmapsets.show.scoreboard.stats.combo'
             div className: "#{bn}__stat-value #{bn}__stat-value--score",
               "#{props.score.max_combo.toLocaleString()}x"
 
-          div className: "#{bn}__stat",
-            div className: "#{bn}__stat-header",
-              hits.header
-            div className: "#{bn}__stat-value #{bn}__stat-value--score",
-              hits.values
+        div className: "#{bn}__stats",
+          for stat in props.hitTypeMapping
+            div
+              key: stat[0]
+              className: "#{bn}__stat"
+              div className: "#{bn}__stat-header",
+                stat[0]
+              div className: "#{bn}__stat-value #{bn}__stat-value--score #{bn}__stat-value--smaller",
+                props.score.statistics["count_#{stat[1]}"].toLocaleString()
 
-          div className: "#{bn}__stat hidden-xs",
+        div className: "#{bn}__stats",
+          div className: "#{bn}__stat",
             div className: "#{bn}__stat-header",
               osu.trans 'beatmapsets.show.scoreboard.stats.misses'
-            div className: "#{bn}__stat-value #{bn}__stat-value--score",
+            div className: "#{bn}__stat-value #{bn}__stat-value--score #{bn}__stat-value--smaller",
               props.score.statistics.count_miss
```

Comment:
might as well add the `toLocaleString()`

Reply:
oh yeah sure for those top ranking scores that have 1000+ misses 🙄

Reply:
and you'll always be in top ranking, right?

Reply:
somehow I doubt there are any maps that exist where you can pass with 1000+ misses?...

Reply:
nf mania?